### ISSUE TITLE
receiving data

### ISSUE THREAD

**Issue by amralabsi** [tpitman/arduino-hm10-unity3d#1]
Hello I was wondering if there is an easy way we can get data from an hm10 device and print it on my mobile device I was trying get coordinates that I can see using BLEscanner to unity.

**Comment by tpitman** [comment]
Yes. If you look at my hm10 example it should give you a good start:

https://github.com/tpitman/arduino-hm10-unity3d

That example shows an example of sending data between 2 hm10, but you can
use one just as well.
